src/prebid.js

```diff
@@ -761,4 +763,29 @@ $$PREBID_GLOBAL$$.getAllWinningBids = function () {
   return $$PREBID_GLOBAL$$._winningBids;
 };
 
+/**
+ * Build master video tag from publishers adserver tag
+ * @param {string} adserverTag default url
+ * @param {object} options options for video tag
+ */
+$$PREBID_GLOBAL$$.buildMasterVideoTagFromAdserverTag = function (adserverTag, options) {
+  var urlComponents = parseURL(adserverTag);
```

Comment:
please add `utils.logInfo('Invoking $$PREBID_GLOBAL$$. buildMasterVideoTagFromAdserverTag', arguments);` line. All public methods should trace their params for debugging purposes.